BlogTemplate/Pages/_Layout.cshtml

```diff
@@ -13,6 +13,7 @@
             <li><a asp-page="/Index">Blog</a></li>
             <li><a asp-page="/About">About</a></li>
             <li><a asp-page="/New">New Post</a></li>
+            <li>@await Html.PartialAsync("_LoginPartial")</li>

```

Comment:
This shows up in a weird place on the page
![settingsplacement](https://user-images.githubusercontent.com/15066077/28852062-592bc6ce-76db-11e7-9ab1-0e312a40da23.PNG)